tests/unit_and_functional/test_k8s_workflow_jobs.py

```diff
@@ -22,7 +22,7 @@
 from datetime import datetime
 from unittest.mock import MagicMock, patch
 
-import kubernetes
+import kubernetes.client as k8s
```

Comment:
Should probably do this across all the tests for the sake of uniformity?

Reply:
Although I must admit, that calling it `k8s` confuses it with our internal `k8s` sub-module, which is why I kept the verbose version. It might me better as `k8s_client`?

Reply:
Yeah I agree, it could easily be confused when flipping between test files from Cli,K8s etc. I've gone with your suggestion.

Reply:
I think you may have forgotten to do this (according to what I'm seeing at this moment in time)?

Reply:
Ignore this - found the right button on GitHub...

Reply:
That's got me a few times as well 🙂